actions/class.UserSettings.php

```diff
@@ -164,6 +175,13 @@ private function getLanguageService(): tao_models_classes_LanguageService
         return tao_models_classes_LanguageService::singleton();
     }
 
+    private function getFeatureFlagChecker(): FeatureFlagChecker
```

Comment:
For better open/close principle, better maintainability

```suggestion
    private function getFeatureFlagChecker(): FeatureFlagCheckerInterface
```